lib/reporter/reporters/minimalJson.js

```diff
@@ -0,0 +1,91 @@
+const lodashPick = require('lodash.pick');
+const path = require('path');
+const Utils = require('../../utils');
+const {Logger} = Utils;
+const BaseReporter = require('../base-reporter.js');
+
+class MinimalJsonReporter extends BaseReporter {
+
+  static isRerunFailedEnabled(argv = {}) {
```

Comment:
no need to have this as separate function, inline this logic.

Reply:
removed 👍